src/pages/Recipes/RecipeEdition/RecipeEdition.jsx

```diff
@@ -0,0 +1,95 @@
+import { yupResolver } from "@hookform/resolvers/yup";
+import React from "react";
+import { useForm } from "react-hook-form";
+import { Route, Routes, useNavigate } from "react-router";
+import { useParams } from "react-router-dom";
+import { v4 as uuidv4 } from "uuid";
+import { Text } from "@/components";
+import AddIngredient from "../AddIngredient/AddIngredient";
+import useRecipe from "../hooks/api/useRecipe";
+import useUpdateRecipe from "../hooks/api/useUpdateRecipe";
+import useIngredients from "../hooks/useIngredients";
+import { schema } from "../RecipeCreation/formSchema";
+import RecipeCreation from "../RecipeCreation/RecipeCreation";
+
+const RecipeEditionRouter = () => {
+  const params = useParams();
+  const recipeId = params.id;
+  const recipeDetails = useRecipe(recipeId).data;
+  const ingredientsList = recipeDetails.ingredients;
+
+  const { mutate, isError, isLoading } = useUpdateRecipe(recipeId);
+  const navigate = useNavigate();
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm({
+    resolver: yupResolver(schema),
```

Comment:
Remember our debugging of that form? 😅  Maybe, it would be more beneficial to rename the names for schemas as well as the filenames to avoid similar confusion in the future? wdyt? Would you be able to adjust these?
```suggestion
    resolver: yupResolver(recipeFormSchema),
```